### ISSUE TITLE
Creation of a WebApp that can take a text input

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#9]
## User story
1. As a user who wants to report a problem/incidents or want to create a service request
2. I want / need a WebApp to input text, that describes my problem
3. So that a ticket that describes my problem is created and I get notified about the creation and the ticket number

## Acceptance criteria
* I can reach the WebApp
* I have a possibility to input my text
* I get a feed back that the creation of the ticket was successful

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
